Debug output Issue

>  Set up jobs
>  Build Sample-account-github-jira/Todo-Application@master27s
>  Run GraphQL query against GH 4.x API
>  Debug output

/home/runner/work/_temp/314346d0-a25a-49d5-870d-07e31135af9a.sh: line 137: syntax error near unexpected token `('
##[error]Process completed with exit code 2.

Could you please share some light on this fix?

--- a/LICENSE
+++ b/LICENSE
@@ -1,0 +1,21 @@
+MIT License
+
+Copyright (c) 2019, 2020 Reload A/S
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in all
+copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+SOFTWARE.

--- a/README.md
+++ b/README.md
@@ -29,6 +29,9 @@ It has some required and some optional settings, which are passed to the action 
 - `JIRA_PROJECT`: The project key for the Jira project where issues should be created, eg `TEST` or `ABC`. (**REQUIRED**)
 - `JIRA_ISSUE_TYPE`: Type of issue to create, e.g. `Security`. Defaults to `Bug`. (*Optional*)
 - `JIRA_WATCHERS`: Jira users to add as watchers to tickets. Separate multiple watchers with comma (no spaces).
+- `JIRA_RESTRICTED_COMMENT_ROLE`: A comment with restricted visibility
+  to this role is posted with info about who was added as watchers to
+  the issue. Defaults to `Developers`. (*Optional*)
 
 Here is an example setup which runs this action every 6 hours.
 

--- a/composer.json
+++ b/composer.json
@@ -7,7 +7,7 @@
         "softonic/graphql-client": "^1.2",
         "symfony/console": "^5",
         "symfony/yaml": "^5.0",
-        "reload/jira-security-issue": "dev-master"
+        "reload/jira-security-issue": "^1.0.0"
     },
     "repositories": [
       {

